### PR TITLE
Add 1.5.30 post

### PR DESCRIPTION
**Pull request by Adam-**

### CODE REVIEW

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+We have been diligently working off the [SotE meta
+issue](https://github.com/runelite/runelite/issues/9459) over the last week, as
+it touches a lot of the plugins and features. Much of it is done and included in
+this release:
+
+- Add divine potion timers to the timers plugin
+- Add Zalcano respawn timer to the boss plugin
+- Add Prifddinas farming patches to the time tracking plugin
+- Add Gauntlet kill count and personal best support to chat commmands
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
- Add Gauntlet kill count and personal best support to chat commands
```

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+- Change "Mark" to "Unmark" when right clicking already marked tiles, objects, and npcs
+- Add option to add skill xptrackers to the canvas on the skill menu tab
+- Fix removing teleblock timer when your opponent who casted the teleblock dies
+- Add total points and points per hour to the NMZ overlay
+- Fix the zoom plugin control reset config value to be properly bounded to
+  valid values
+- Add 2005 style sprite override for friend/ignore list toggle icon
+- The item charges plugin now supports Rings of Forging
+- Add gnome stronghold fishing spots to fishing plugin
+- Incrase smelting overlay timeout to account for cannonballs
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
- Increase smelting overlay timeout to account for cannonballs
```

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+- Show grand exchange value when examining items in the seed vault
+- The quest list plugin now has an additional filter to show uncompleted quests
+  only
+- Add support for Lava Eels to the fishing plugin
+- Fix world map quest marker for Depths of Despair
+- Add Hespori to the loot tracker
+- The item prices plugin can now show HA profit when casting on items
+- Change "Mark" to "Unmark" when right clicking already marked tiles, objects, and npcs
+- Add option to add skill xptrackers to the canvas on the skill menu tab
+- Fix removing teleblock timer when your opponent who casted the teleblock dies
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
- Fix removing teleblock timer when your opponent who cast the teleblock dies
```

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+  timers
+- Add Prifddinas and surrounding areas to the skybox plugin
+- Add Prifddinas and surrounding areas to the possible Discord locations
+- Add crystal sapling to the item identification plugin
+- Add new crystal tools to the idle notifier
+- Add crystal axe to woodcutting plugin and wintertodt plugin
+- Add crystal pickaxe to motherlode plugin
+- Add crystal implings to the impling plugin
+
+Item despawn timers have been added to the ground items plugin, which shows how
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
Item despawn timers have been added to the ground items plugin, which show how
```

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+long until an item will despawn for items which you drop and receive as loot
+from a PvM kill.
+
+![despawntimer](/img/blog/1.5.30-Release/despawntimer.png)
+
+The world hopper plugin now has an option to show the current worlds ping
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
The world hopper plugin now has an option to show the current world's ping
```

or `s/worlds/world/`

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+
+![ping](/img/blog/1.5.30-Release/ping.png)
+
+The clue plugin now shows locations for emote clues on the overlay, instead of
```
Comment (Nightfirecat) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]: I think this reads better as "The clue plugin now shows locations for emote clues on the overlay in addition to on the world map".

File: src/_posts/2019-08-01-10-00-1.5.30-Release.md [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+- Add Zalcano respawn timer to the boss plugin
+- Add Prifddinas farming patches to the time tracking plugin
+- Add Gauntlet kill count and personal best support to chat commands
+- Add Prifddinas agility course "kill count" and personal best to chat commands
+- Updates to the world map:
+  - Add SotE quest start
+  - Add rare tree, transportation, and shortcut tooltips
+  - Fix Elf camp teleport location
+  - Add farming patch tooltips
+  - Fix teleport crystal location
```
Comment (Alexsuperfly) [inline src/_posts/2019-08-01-10-00-1.5.30-Release.md]:
```suggestion
  - Add teleport crystal locations
```

They weren't there to begin with

**Pull request merged**